### PR TITLE
Disable tests blocking CI

### PR DESCRIPTION
**Pull request by hoyosjs**
Disables tests for #55271 and #55435

### CODE REVIEW

**Review by safern — APPROVED** [review]
LGTM. cc: @wfurt

File: src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostByNameTest.cs [hunk 1]
```diff
@@ -112,8 +113,9 @@ public void DnsObsoleteGetHostByName_EmptyString_ReturnsHostName()
             Assert.Contains(Dns.GetHostName(), entry.HostName, StringComparison.OrdinalIgnoreCase);
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/1488", TestPlatforms.OSX)]
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotArm64Process), nameof(PlatformDetection.IsThreadingSupported))] // [ActiveIssue("https://github.com/dotnet/runtime/issues/27622")]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/1488", TestPlatforms.OSX)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/55271", TestPlatforms.Linux)]
```
Comment (wfurt) [inline src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostByNameTest.cs:118]: I think the DNS tests are environmental. WE should disable them only on failing distro.

File: src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostEntryTest.cs [hunk 2]
```diff
@@ -21,11 +21,12 @@ public async Task Dns_GetHostEntryAsync_IPAddress_Ok()
         }
 
         [ActiveIssue("https://github.com/dotnet/runtime/issues/1488", TestPlatforms.OSX)]
-        
+
         [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotArm64Process))] // [ActiveIssue("https://github.com/dotnet/runtime/issues/27622")]
         [InlineData("")]
         [InlineData(TestSettings.LocalHost)]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/51377", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/55271", TestPlatforms.Linux)]
```
Comment (wfurt) [inline src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostEntryTest.cs:29]: same as above. If that fails only on SLES we should disable them selectively.

File: src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostByNameTest.cs (hunk 1 above)
Comment (hoyosjs) [reply src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostByNameTest.cs:118]: I don't see a way to disable in SLED.

Comment (hoyosjs) [reply src/libraries/System.Net.NameResolution/tests/FunctionalTests/GetHostByNameTest.cs:118]: I think I see how.

**Comment by steveisok** [comment]
The mono linux x64 failure looks like a flake.

**Comment by hoyosjs** [comment]
Yeah, `Connect_DualMode_DnsConnect_RetrievedEndPoints_Success` was under investigation. I know @antonfirsov changed something in this test recently to try to remove the flakiness.

**Comment by ManickaP** [comment]
Related https://github.com/dotnet/core-eng/issues/13675
This seems like an infra issue.

**Pull request merged**